ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/productdetails.html

```diff
@@ -1,9 +1,18 @@
 <div class='product-details-cmp'>
 	<div class='product-details-cmp__img aem-Grid aem-Grid--9 aem-Grid--default--12'>
 		<img class='product-details-cmp--product-img' src="${properties.fileReference}" alt="image" />
-	</div>
-	<div class='product-details-cmp__product-details aem-Grid aem-Grid--3 aem-Grid--default--12'> <b class='product-details-cmp__prduct-name'>${properties.productName} </b>
-		<p class='product-sku'>${properties.productSku}</p>
+	<div>
+            <div class="productName">
+                <b class='product-details-cmp__prduct-name'>${properties.productName} </b>
+            </div>
+               <div data-sly-resource="${'SocialShare' @ resourceType='hclecomm/components/content/SocialShare', decoration=true}"></div>
```

Comment:
@uditnaagar777 plz remove share component  from pdp component as we can author it beside title component as separate component

Reply:
As of now , we dont have title component. We have created a seperate component for social share. Once we get the title component & there is any UI issue, we will update CSS accordingly